fnv1/hash32.go

```diff
@@ -0,0 +1,54 @@
+package fnv1
+
+const (
+	// FNV-1
+	offset32 = uint32(2166136261)
+	prime32  = uint32(16777619)
+
+	// Init32 is what 64 bits hash values should be initialized with.
```

Comment:
This comment seems misleading, did you mean 32 bits?

Reply:
oh, sorry, will fix it now